velox/docs/functions/presto/math.rst

```diff
@@ -255,9 +255,14 @@ Probability Functions
     The successProbability must be real value in [0, 1], numberOfTrials and value must be
     positive integers with numberOfTrials greater or equal to value
 
+.. function:: inverse_beta_cdf(a, b, p) -> double
+
+    Compute the inverse of the Beta cdf with given a, b parameters for the cumulative
+    probability (p): P(N < n). The a, b parameters must be positive real values.
```

Comment:
Nit : The prestodb documentation also says "all of type DOUBLE" here. Please add that.

Also the documentation has 2 separate categories of Probability functions : cdf and Probability functions : inverse_cdf. Lets follow the same since these could be a long list of functions.

Reply:
Addressed in latest commit.